includes/class-content-toc.php

```diff
@@ -113,7 +113,7 @@ public function shortcode( $atts, $content = null ) {
 						'<li class="hm-toc-item-%s"><a href="#heading-%d">%s</a></li>',
 						esc_attr( $item[2] ),
 						esc_attr( $key_current ),
-						esc_html( $item[1] )
+						esc_html( strip_tags( $item[1] ) )
```

Comment:
Do I need to alter values I pass to filter or shall I pass the original ones? Or add another value with stripped tags?